backend/de.metas.adempiere.adempiere/base/src/main/java-gen/org/compiere/model/I_C_Order.java

```diff
@@ -1442,7 +1442,7 @@ public interface I_C_Order
 	String COLUMNNAME_EMail = "EMail";
 
 	/**
-	 * Set External ID.
+	 * Set SAP PayT ID.
```

Comment:
I'm not sure this is correct. Why did the vanilla AD_Element associated with ExternalId change?

Reply:
corrected